terraform/modules/rds-serverless/main.tf

```diff
@@ -0,0 +1,90 @@
+provider "aws" {
+  region = "ap-southeast-2"
+}
+
+######################################
+# Data sources to get VPC and subnets
+######################################
+# https://registry.terraform.io/providers/hashicorp/aws/latest/docs/data-sources/vpc
+data "aws_vpc" "da-c02-vpc" {
+  #id = "${var.vpc_id}"
+  id = "vpc-02ec2836691642ace"
+}
+
+data "aws_subnet_ids" "da-c02-vpc" {
+  vpc_id = data.aws_vpc.da-c02-vpc.id
+}
```

Comment:
No need to find the VPCs, that should come as a variable from the "networking" module.